code/modules/mob/dead/observer/observer.dm

```diff
@@ -16,6 +16,7 @@ GLOBAL_VAR_INIT(observer_default_invisibility, INVISIBILITY_OBSERVER)
 	canmove = FALSE
 	alpha = 127
 	move_resist = INFINITY	//  don't get pushed around
+	anchored = 1 // don't get moved by mass drivers
```

Comment:
You can change that to `anchored = TRUE`, since it's a boolean value.
`TRUE` does the exact same thing as `1`, but it's easier to read.

Reply:
my bad, so used to messing with view variables where everything generally tends to be 0 or 1, ahaha. still, will keep that in mind for the future.

Reply:
The line above, `move_resist = INFINITY`, already covers stuff like this, and its the mass driver code's fault for not checking for move resist before pushing objects. I suggest adding a check on the mass driver for this.

Reply:
What steel said

Reply:
What steel said

Reply:
isnt it easier to just modify this one file for that tho?